server/ntb/macros/nb_NO_to_nn_NO_metadata_macro.py

```diff
@@ -18,6 +20,15 @@ def nb_NO_to_nn_NO_metadata_macro(item, **kwargs):
             'scheme': None
         }
     ]
+    """
+    Translation start
+    """
+    payload = {k: item.get(k) for k in item if k in ('guid', 'evolvedfrom', 'versioncreated', 'headline', 'description_text', 'description_html', 'body_text', 'body_html')}
+    r = requests.post('http://api.smalldata.no:8080/translate', data=payload, auth=HTTPBasicAuth('superdesk', 'babel'))
+    item.update(r.json())

```

Comment:
would make sense imo to check the response code, in case there would be some errors in reaching the service etc:
```python3
if r.status_code == 200:
    item.update(r.json())
```

Reply:
Yes, I will add that. I should also set a request timeout here. I'm thinking around 10 seconds, although I have to test this a little bit first before deciding on a value.

Reply:
you can set specific timeout for connection and then for service, like `(10, 30)` so it would timeout after 10s if it can't reach the service and after another 30 if the service doesn't respond